### PR TITLE
Uuid clarification

### PR DESCRIPTION
**Pull request by amencarini**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,8 +76,9 @@ A subscriber can "catch up" event if it hasn't pulled events for a while
 
 Environment variables:
 
-* `ROUTEMASTER_CLIENTS` - the allow UUIDs
+* `ROUTEMASTER_CLIENTS` - the allow UUIDs, separated by commas
```
Comment (nas) [inline README.md]: `s/allow UUIDs/allowed UUIDs` since you are touching it :wink:

Comment (amencarini) [inline README.md]: well spotted

**Comment by nas** [comment]
> Grammar Naszi fix

Please  fix your comment spelling.

jonking :wink:

**Comment by amencarini** [comment]
Fenks!

**Pull request merged**